infrastructure/README.md

```diff
@@ -18,7 +18,52 @@ KIT Infrastructure creates a base K8s cluster with below  add-ons by default  bu
 - Flux v2
 - Kubernetes Iteration Toolkit (KIT) Operator (optional)
 
-Flux is setup, by deafult, to monitor the KIT git repo path `./infrastructure/k8s-config/clusters/kit-infrastructure`, which includes other add-ons that do not require AWS credentials such as tekton, prometheus, grafana, and the metrics-server. 
+Flux is setup, by deafult, to monitor the KIT git repo path `./infrastructure/k8s-config/clusters/kit-infrastructure`, which includes other add-ons that do not require AWS credentials such as tekton, prometheus, grafana, the metrics-server and [perf dash].
+
+#### Perf Dash
+
+Perf dash is optional. If you want to disable it, you can simply remove or comment out the content in `./infrastructure/k8s-config/clusters/kit-infrastructure/perfdash/perfdash.yaml`.
```

Comment:
Doesn't provide info if its enabled by default? 
> If you want to disable it, you can simply remove or comment out the content in

I don't think this will work because it will be synced from Flux and we don't want everyone to be changing KIT repo for disabling perfdash. 

Also enabling by default would mean existing clusters in Karpenter will get this installed which might be a surprise for them. I would minimize such changes in this repo or at least get a consent from Karpenter team.
cc @bwagner5

Reply:
Thanks for bringing this up! It seems karpenter do use KIT for testing: https://github.com/aws/karpenter/tree/main/test/infrastructure#setup.

>I don't think this will work because it will be synced from Flux and we don't want everyone to be changing KIT repo for disabling perfdash.

I thought a user cloud have their own forks where they can make some changes. I can rephrase it to make it more clear. If you don't think that's a valid assumption, I can change it.

>Also enabling by default would mean existing clusters in Karpenter will get this installed which might be a surprise for them.

I guess it won't be a surprise for Karpenter team, since it doesn't break their cluster. But I guess this addon is not useful for them, since they don't CL2 tests at all. We may not want to install it default.
An alternative is to add it as an addon under `./infrastructure/lib/addons/` using typescript. A flag can be used to gate it and make it disabled by default.
@prateekgogia @hakuna-matatah WDYT?

Reply:
Current state of things - We install certain addons today through typescript cdk thing if that addon requires aws resources to function otherwise we install them through flux. Customers have an option of disabling certain addons through cdk if they are being installed but not through flux.
We need to take a stand on whether we want to move all the addons to be installed through type-script and give customers an option of disabling unwanted addons (OR) have a separate directory or git repo to be pointed out for all optional addons so customers have flexibility to choose at the time of setup. For the latter one (i.e; flux route), all we have to do is basically send an array list here for repoPaths and repos, so customers can pass it at the time of setup  - https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/infrastructure/lib/kit-infrastructure.ts#L167-L177

Reply:
> have a separate directory or git repo to be pointed out for all optional addons so customers have flexibility to choose at the time of setup.

I guess this approach makes more sense.